api/authenticators/okta/saml.go

```diff
@@ -0,0 +1,84 @@
+package okta
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/riotgames/key-conjurer/api/core"
+
+	"golang.org/x/net/html"
+)
+
+func walkHTMLTree(node *html.Node, condition func(*html.Node) bool) (*html.Node, bool) {
```

Comment:
In order to make this work properly we need to reach into the HTML to find the SAML Response from Okta. The XML parser won't work for this, unfortunately.